sentry_sdk/tracing.py

```diff
@@ -54,7 +95,7 @@ def continue_from_headers(cls, headers):
         parent = cls.from_traceparent(headers.get("sentry-trace"))
         if parent is None:
             return cls.start_trace()
-        return parent.new_span()
+        return parent.new_span(same_process_as_parent=False)
```

Comment:
`continue_from_environ` and `continue_from_headers` seems like helper functions that should exist here.
Like coupling these http/python environ stuff with the basic tracing API.

Reply:
Decided against this, I consider those to be constructors just like `__init__`. I think it's fine if we don't stabilize this aspect of the API if that's what you're after.